src/config/rendering.ts

```diff
@@ -0,0 +1,155 @@
+export interface ClusteringConfig {
+  monitor: boolean;
+  mode: string;
+  maxConcurrency: number;
+  timeout: number;
+}
+
+// https://chromedevtools.github.io/devtools-protocol/tot/Network/#method-emulateNetworkConditions
+type NetworkConditions = {
+  offline: boolean;
+  downloadThroughput: number;
+  uploadThroughput: number;
+  latency: number;
+};
+
+export interface RenderingConfig {
+  chromeBin?: string;
+  args: string[];
+  ignoresHttpsErrors: boolean;
+  timezone?: string;
+  acceptLanguage?: string;
+  width: number;
+  height: number;
+  deviceScaleFactor: number;
+  maxWidth: number;
+  maxHeight: number;
+  maxDeviceScaleFactor: number;
+  pageZoomLevel: number;
+  mode: string;
+  clustering: ClusteringConfig;
+  verboseLogging: boolean;
+  dumpio: boolean;
+  timingMetrics: boolean;
+  headed?: boolean;
+  networkConditions?: NetworkConditions;
+  emulateNetworkConditions: boolean;
+}
+
+export const defaultRenderingConfig: RenderingConfig = {
+  chromeBin: undefined,
+  args: ['--no-sandbox', '--disable-gpu'],
+  ignoresHttpsErrors: false,
+  timezone: undefined,
+  acceptLanguage: undefined,
+  width: 1000,
+  height: 500,
+  headed: false,
+  deviceScaleFactor: 1,
+  maxWidth: 3000,
+  maxHeight: 3000,
+  maxDeviceScaleFactor: 4,
+  pageZoomLevel: 1,
+  mode: 'default',
+  clustering: {
+    monitor: false,
+    mode: 'browser',
+    maxConcurrency: 5,
+    timeout: 30,
+  },
+  emulateNetworkConditions: false,
+  verboseLogging: false,
+  dumpio: false,
+  timingMetrics: false,
+};
+
+export function populateRenderingConfigFromEnv(config: RenderingConfig, env: NodeJS.ProcessEnv, isPlugin: boolean) {
```

Comment:
Good idea, I updated this! I just changed it a bit because I find it more convenient to see all config fields for one mode next to each other rather than alternating between server field and plugin field. 

Plus, with this, we can't really support env variables in one mode only so both lists should contain the same fields.